daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/TransactionCoder.scala

```diff
@@ -350,30 +341,21 @@ object TransactionCoder {
           children <- decodeChildren(protoRollback.getChildrenList)
         } yield ni -> Node.Rollback(children)
       case NodeTypeCase.CREATE =>
-        val protoCreate = protoNode.getCreate
         for {
           ni <- nodeId
-          pkgName <- decodePackageName(protoCreate.getPackageName)
-          c <- ValueCoder.decodeCoid(protoCreate.getContractId)
-          tmplId <- ValueCoder.decodeIdentifier(protoCreate.getTemplateId)
-          arg <- ValueCoder.decodeValue(nodeVersion, protoCreate.getArgUnversioned)
-          stakeholders <- toPartySet(protoCreate.getStakeholdersList)
-          signatories <- toPartySet(protoCreate.getSignatoriesList)
-          keyOpt <- decodeOptionalKeyWithMaintainers(
-            nodeVersion,
-            tmplId,
-            protoCreate.getKeyWithMaintainers,
+          createNode = protoNode.getCreate
+          _ <- Either.cond(
+            createNode.getCreatedAt == 0L,
+            (),
+            DecodeError("unexpected create_at field in createNode"),
```

Comment:
```suggestion
            DecodeError("unexpected created_at field in create node"),
```

Reply:
(As far as I can tell, there is no `createNode` in the proto: just a `create` field in a oneof in the `Node` message.)